dashboard/app/batch_reproexport.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"net/http"
+
+	"google.golang.org/appengine/v2"
+	"google.golang.org/appengine/v2/log"
+)
+
+const exportTimeoutSeconds = 10000 * 2 // upstream has apx 7k reproducers, 1s each max (throttling)
```

Comment:
It's not expected to have throttling if we merge the other PR, right?
The comment may be confusing for future readers as they will assume we do have throttling for our export.